backend/inspirehep/search/errors.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from inspirehep.errors import BaseRestError
+
+
+class MaximumSearchPageSizeExceeded(BaseRestError):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.description = f"Maximum search page size exceeded."
```

Comment:
This is not useful, you should say what the maximum is.

Reply:
:+1: